### ISSUE TITLE
Add dynamic parameters to dynamic actions

### ISSUE THREAD

**Issue by vbnogueira** [p4tc-dev/linux-p4tc-pub#32]
Change the `act` command to accept parameters instead of requiring an action instance
This means, allow arguments to be processed and passed in runtime instead of having them always preprocessed in init time.

For example instead of always doing something like this:

```
$TC actions add action simple_l3_pna/mirror_to_port param port type dev dummy1 index 13

$TC p4template create action/simple_l3_pna/my_act \
cmd act simple_l3_pna.mirror_to_port.1
```

Do:

```
$TC p4template create action/simple_l3_pna/my_act \
cmd act simple_l3_pna.mirror_to_port dev.dummy1

```

**Comment by jhsmt** [comment]
Should the description be using p4template and create instead?
...
$TC p4template create action/simple_l3_pna/my_act \
cmd act simple_l3_pna.mirror_to_port.1

...
$TC p4template create action/simple_l3_pna/my_act \
cmd act simple_l3_pna.mirror_to_port param port dummy1

**Comment by vbnogueira** [comment]
Closed with #45
